packages/frontend/craco.config.js

```diff
@@ -40,28 +36,14 @@ module.exports = {
                 },
             };
             if (env !== "production") return config;
-            config.optimization = {
-                ...config.optimization,
-                moduleIds: "deterministic",
-                runtimeChunk: "single",
-            };
-            config.output = {
-                ...config.output,
-                publicPath: "auto",
-                filename: (pathData) => {
-                    return pathData.chunk.name === "sw"
-                        ? "sw.js"
-                        : "[name].[contenthash:8].js";
-                },
-                chunkFilename: "[name].[contenthash:8].js",
-                assetModuleFilename: "[name].[contenthash:8][ext]",
-            };
-            // TODO: make Workbox and precaching work
-            // config.plugins.push(
-            //     new WorkboxWebpackPlugin.InjectManifest({
-            //         swSrc: join(__dirname, "/src/sw.ts"),
-            //     })
-            // );
+            // TODO: make Workbox and precaching work (all
```

Comment:
We can remove the TODO since we are using `Workbox` now, right?

Reply:
Nono, right now we're basically only using that plugin to correctly transpile the service worker, but we're not really leveraging Workbox (that will be done when we enable precaching and turn Carrot into a proper PWA).